hbase/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,13 @@
+# Settings to quiet third party logs that are too verbose
```

Comment:
It would be interesting to uncomment logs to be sure that the comment above about reducing connections number worked.